doc/api/esm.md

```diff
@@ -147,15 +147,13 @@ be written:
 import url from 'url';
 import path from 'path';
 import process from 'process';
+import module from 'module';
```

Comment:
maybe capital `Module`

Reply:
Only saw this now for some reason, sounds sensible!